### ISSUE TITLE
Accessibility review of Header/Footer designs

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#14339]
## Description
The new header/footer designs need go through an initial accessibility review in order to make sure there are no potential areas that could be problematic when we move into development.

The views that will be audited are:

- Full header
- Full footer
- Skinny header (form experience, check-in, etc.)

*The skinny footer is out of scope for this review and will be audited when it is available as part of a separate ticket.

### Relevant links

- [Sketch file](https://www.sketch.com/s/4558672b-741f-4d59-84f0-796020d3ef57)
- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/header-footer/federal-standardized-header-footer)

## Additional Context
Documentation should include:
- Questions regarding the designs (items that aren't clear that would impact that accessibility)
- Any required changes to current designs
- Required accessibility markup that will be needed for development

## Acceptance Criteria
- [x] Design mockups have been reviewed by accessibility specialist
- [x] Feedback has been documented
- [x] Share feedback with stakeholders

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [x] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by laflannery** [comment]
Dave C will follow up to confirm:
- Accessibility item links for the footer
- Who will be  moving the work forward (Ryan or Amanda) and who I should be sending my recommendations to/meeting with

**Comment by laflannery** [comment]
Scheduled meeting to review feedback with Ryan and Amanda for Tuesday 8/1
